AGENTS.md

```
@@ -1,8 +1,27 @@
 # Repository Guidelines
 
+## Project Overview
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)